### ISSUE TITLE
Update README, docs, ci

### ISSUE THREAD

**Issue by dpo** [JuliaSmoothOptimizers/QuadraticModelsCPLEX.jl#11]
@geoffroyleconte 

Then we could make a first release?

**Comment by geoffroyleconte** [comment]
Yes, I only need to add the presolve to a mps file before. Will the CI work? Because a license is required I think

**Comment by dpo** [comment]
@MonssafToukal we need your magic here and in two other similar repos.

**Comment by MonssafToukal** [comment]
Ok, no problem. I will setup the unit tests for this repository!

**Comment by dpo** [comment]
@geoffroyleconte Could you please set up documentation here and a basic tutorial?

**Comment by geoffroyleconte** [comment]
documentation here: https://github.com/JuliaSmoothOptimizers/QuadraticModelsCPLEX.jl/pull/19
